packages/2019-disaster-game/src/components/atoms/NewBadge.js

```diff
@@ -59,70 +54,68 @@ const titleText = css`
   font-size: 14rem;
 `;
 
-const NewBadge = ({ type, badges, playAudio, stopAudio }) => {
-  const [badgeInfo, setBadgeInfo] = useState(badges[type]);
+const NewBadge = ({
+  badgeData,
+  displayBadgeTime = 11,
```

Comment:
Should this timing be pulled out to live with other timing variables?

Reply:
I think for a default value it's ok?

Reply:
Agreed